OasisDEX/oasis-borrow
# Overview page fixes

# Overview page fixes
  
## Changes 👷‍♀️

- [Includes liquidity on overview page fro DPM positions.](https://app.shortcut.com/oazo-apps/story/7209/overview-liquidity-is-not-complete)
- [Shows AAVE positions correctly on overview page](https://app.shortcut.com/oazo-apps/story/7263/bug-routing-issue-with-aave-from-my-positions-page).  Consolidates aaveDpmPositions and aavePositions into one observable.
  
## How to test 🧪

- open positions overview page with an AAVE stETH/ETH postiion.
- ensure that liquidity is displayed and the correct format and value
- ensure that multiply positions created using DS Proxy show as multiply positions on overview page
